Adding new concepts and procedures for Multi CV hosts and AKs (#3609)
theforeman/foreman-documentation: guides/common/modules/proc_viewing-multiple-content-view-environment-activation-keys.adoc

[id="viewing-multiple-content-view-environment-activation-keys"]
= Viewing multiple content view environment activation keys

Use this procedure to view activation keys with multiple content view environments, or multi-environment activation keys, in the {ProjectWebUI} and Hammer CLI.

.Prerequisites
* Ensure you have activation keys created.

> For more information, see xref:Creating_an_Activation_Key_{context}[Creating an activation key].

Reviewer: ```suggestion
For more information, see xref:Creating_an_Activation_Key_{context}[].
```

suggestion: drop title. xrefs do not need titles. Omitting them makes it easier to maintain the docs.

.Procedure
. In the {ProjectwebUI}, navigate to *Content* > *Lifecycle* > *Activation Keys*.      
. Select an activation key.
. Under the *Details* tab, the *Content view environments* are displayed.

[NOTE]
====

> You cannot reassign a multi-environment activation key's content view environments using the web UI.

Reviewer: ```suggestion
You cannot reassign a multi-environment activation key's content view environments using the {ProjectWebUI}.
```

To change content view environments, update the activation key with Hammer.
====

.CLI procedure

> . Run the following command to view a list of activation keys in your organization including single environment and multi-environment keys:

Reviewer: ```suggestion
. View a list of activation keys in your organization including single environment and multi-environment keys:
```

+
[options="nowrap" subs="+quotes"]
----
$ hammer activation-key list --organization-id _My_Organization_ID_
----

> The `CONTENT VIEW ENVIRONMENTS` column displays the labels of content view environments associated with each activation key. The `MULTI CONTENT VIEW ENVIRONMENT` column shows `Yes` for multi-environment activation keys.

Reviewer: ```suggestion
The `CONTENT VIEW ENVIRONMENTS` column displays the labels of content view environments associated with each activation key.
The `MULTI CONTENT VIEW ENVIRONMENT` column shows `Yes` for multi-environment activation keys.
```

+
. Run the following command to view multi-environment information for a single activation key:
+
[options="nowrap" subs="+quotes"]
----
$ hammer activation-key info --id _My_Activation_Key_ID_
----

> The information for the activation key is displayed including multiple content view environments.

Reviewer: ```suggestion
+
The information for the activation key is displayed including multiple content view environments.
```

> [id="Viewing_multiple_content_view_environment_activation_keys_{context}"]

Reviewer: ```suggestion
[id="viewing-multiple-content-view-environment-activation-keys"]
```

= Viewing multiple content view environment activation keys

> Use this procedure to view activation keys with multiple content view environments or multi-environment activation keys in the {ProjectWebUI} and Hammer CLI.

Reviewer 2: ```suggestion
Use this procedure to view activation keys with multiple content view environments, or multi-environment activation keys, in the {ProjectWebUI} and Hammer CLI.
```

Comma makes a difference to meaning here! (also, should multi-environment activation keys be first?)

Author: I'll put "multi-environment activation key"s first. Commas aren't necessary since the phrase "or multi-environment activation keys" is essential to the sentence.

.Prerequisites
* Ensure you have activation keys created.
For more information, see xref:Creating_an_Activation_Key_{context}[Creating an activation key].

.Procedure
. In the {ProjectwebUI}, navigate to *Content* > *Lifecycle* > *Activation Keys*.      
. Select an activation key.
. Under the *Details* tab, the *Content view environments* are displayed.

[NOTE]
====
You cannot reassign a multi-environment activation key's content view environments using the web UI.

> To change a multi-environment activation key's content view environments, update the activation key with Hammer.

Reviewer 2: While technically accurate, I think it's too much of a mouthful and we could remove some words:

```suggestion
To change content view environments, update the activation key with Hammer.
```

====

.CLI procedure
. Run the following command to view a list of activation keys in your organization including single environment and multi-environment keys:
+
[options="nowrap" subs="+quotes"]
----

> $ hammer activation-key list --organization _“Default Organization”_

Reviewer: ```suggestion
$ hammer activation-key list --organization-id _My_Organization_ID_
```

> $ hammer activation-key list --organization “Default Organization”

Reviewer 2: italicize example values

----
The `CONTENT VIEW ENVIRONMENTS` column displays the labels of content view environments associated with each activation key. The `MULTI CONTENT VIEW ENVIRONMENT` column shows `Yes` for multi-environment activation keys.
+
. Run the following command to view multi-environment information for a single activation key:
+
[options="nowrap" subs="+quotes"]
----

> $ hammer activation-key info --id _[1,2,3]_

Reviewer 2: `--id` is just a single number, not an array.

I see elsewhere in the docs we do it like this:

```suggestion
$ hammer activation-key info --id _My_Activation_Key_ID_
```

> $ hammer activation-key info --id

Reviewer 2: This one is missing an example value

----
The information for the activation key is displayed including multiple content view environments.

> 
> 

Reviewer: ```suggestion
```

[id="Viewing_multiple_content_view_environment_activation_keys_{context}"]
= Viewing multiple content view environment activation keys

> You can view multiple content view environment activation keys in the {ProjectWebUI} and the Hammer CLI.

Reviewer 2: Sounds good to me!  (The four-word adjective is a lot, though.. maybe `activation keys with multiple content view environments, or multi-environment activation keys`  (and the same for hosts, naturally.))

To change this activation key's content view environments, update the activation key with Hammer.
====

.CLI procedure
. Connect to {ProjectServer} using your SSH. 
. Run the following command to see a list of content view environments:
+
[options="nowrap" subs="+quotes"]
----
$ hammer content-view-environment list --organization “Default Organization”
----
This displays a list of available content view environments.
+
. Run the following command to create a multiple content view environment activation key:
+
[options="nowrap" subs="+quotes"]
----
$ hammer activation-key create --name ”My_Activation_Key”  - -organization “Default Organization” - -content-view-environments “Content_View_Environment_1”,“Content_View_Environment_2”
----
+
. Run the following command to view multiple content view environment activation key organizations:
+
[options="nowrap" subs="+quotes"]
----
$ hammer activation-key list --organization “Default Organization”
----
The multiple content view activation key organizations are displayed.
+
. Run the following command to view multiple content view environment activation key information:
+
[options="nowrap" subs="+quotes"]
----

> $ hammer activation-key info --_”Information_ID”_

Reviewer 2: ```suggestion
$ hammer activation-key info --_”Information_ID”_
```
Is this a placeholder? Should be --name or --id
